index.html

```diff
@@ -87,3 +86,299 @@ <h4 class="sub-title">I'm a software developer</h4>
     </main>
   </body>
 </html>
+
+
+      <!-- My works begins here -->
+
+      <section class="my-works">
+        <h2 class="section-title">
+          My Recent Works
+          <hr class="horizontal-line" />
+        </h2>
+
+        <div class="works-outer-wrapper">
+          <ul class="works-inner-wrapper">
+            <li>
+              <div class="first-card">
+                <img
+                  class="first-work"
+                  src="images/first_work.svg"
+                  alt="First Work"
+                />
+                <div class="card-content">
+                  <h3 class="card-title">Multi-Post Stories</h3>
+                  <p class="card-description">
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard dummy text ever since the 1500s, when an unknown
+                    printer took a standard dummy text.
+                  </p>
+                  <ul class="card-list">
+                    <li class="card-list-item">css</li>
+                    <li class="card-list-item">html</li>
+                    <li class="card-list-item">bootstrap</li>
+                    <li class="card-list-item">Ruby</li>
+                  </ul>
+                  <a class="card-link" href="#">See Project</a>
+                </div>
+              </div>
+            </li>
+
+            <li>
+              <div class="second-card">
```

Comment:
- [ ] As in the project requirements you must stick to the design as much as possible (e.g., font, colors, images, text, margins). Kindly make sure the space between your word is the same, and the fonts(font-size, font-family, font-weight, font-style,...) are the same🙃 🙃 

Your design

<img width="271" alt="Screen Shot 2022-10-25 at 21 12 17" src="https://user-images.githubusercontent.com/74501523/197861791-32bbbec2-5f00-4232-abd9-8203f823d7fd.png">

Figma design

<img width="288" alt="Screen Shot 2022-10-25 at 21 12 38" src="https://user-images.githubusercontent.com/74501523/197861840-42397a68-026a-4878-b22a-f1c661c1dc69.png">

Reply:
Fixed.